apps/inbound-mail/src/server/queue-service.ts

```diff
@@ -9,14 +9,8 @@ export class QueueService {
   private bullConfig: QueueBaseOptions = {
     connection: {
       db: Number(process.env.REDIS_DB_INDEX) ?? 2,
```

Comment:
I do think we use the same values also for the Inbound Mail Queue Service but at first touch I was conservative and left this same injection and just moved the shared values with the other apps implementations.